docs/async/index.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: default
+title: Asynchronous Search
+nav_order: 51
+has_children: true
+---
+
+# Asynchronous Search
+
+Searching large volumes of data can take a long time, especially if it’s across warm nodes or multiple remote clusters.
+
+Asynchronous search lets you run search requests that execute in the background. You can monitor the progress of these searches and get back partial results as they become available. After the search completes, you can save the results to examine at a later time.
+
+Asynchronous search requires Elasticsearch 7.10.2 or higher.
```

Comment:
This is common for the AES documentation and I guess fine to keep here, but for the ODFE plugins, we typically just use the compatibility table: https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#plugin-compatibility